config/src/config/consensus_config.rs

```diff
@@ -62,12 +62,12 @@ impl Default for ConsensusConfig {
             // defaulting to under 0.5s to broadcast the proposal to 100 validators
             // over 1gbps link
             max_sending_block_bytes: 600 * 1024, // 600 KB
-            max_sending_block_bytes_quorum_store_override: 10 * 1024 * 1024, // 5MB
-            max_receiving_block_txns: 16000,
+            max_sending_block_bytes_quorum_store_override: 10 * 1024 * 1024, // 10MB
+            max_receiving_block_txns: 1.5 * MAX_SENDING_BLOCK_TXNS_QUORUM_STORE_OVERRIDE,
```

Comment:
Does this compile? Actually I must have commented the wrong thing. This should just be whatever works with `max_sending_block_txns: 2500`

Reply:
Thanks, you are right. Reverted this change.